components/form-builder/app/Template.tsx

```diff
@@ -46,11 +46,13 @@ export const PageTemplate = ({
   title,
   navigation,
   leftNav = true,
+  autoWidth = false,
 }: {
   children: React.ReactNode;
   title: string;
   navigation?: React.ReactElement;
   leftNav?: boolean;
+  autoWidth?: boolean;
```

Comment:
Change is to allow the download table top nav links and the download table to go beyond the Form-builder fixed size